qiita_pet/support_files/doc/source/analyzingsamples/index.rst

```diff
@@ -512,3 +512,526 @@ Beta Correlation
   * **Pearson** :ref:`[56]<reference56>` : Measures how strong the linear relationship is between 2 variables
 
 * Gives scatterplot of the distance matrix on the x-axis and the variable being tested on the y-axis
+
```

Comment:
I think this should be it's own document/section with a title like: Introduction to download and reanalyze Qiita data, or something like that